crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/controller/CrownCourtProceedingControllerTest.java

```diff
@@ -145,4 +150,15 @@ void givenAValidInput_whenUpdateIsInvoked_RequestObjectFailsValidation() throws
                         HttpMethod.PUT, updateApplicationRequestJson, ENDPOINT_URL + "/update-crown-court"))
                 .andExpect(status().is4xxClientError());
     }
+
+    @Test
+    void givenAnInput_whenUpdateIsInvokedAndValidationFails_thenBadRequestResponseIsReturned() throws Exception {
+        var apiUpdateApplicationRequest =
```

Comment:
Just checking if the variable declaration was intentional here (apiUpdateApplicationRequest as var) This is always going to be ApiUpdateApplicationRequest.

Reply:
No intention. Just to keep it consistent with the other test cases in CrownCourtProceedingControllerTest.java